configs/local.config

```diff
@@ -1,26 +1,23 @@
 process.executor = 'local'
```

Comment:
just to get it right for my understanding:
container.config includes just the container(where to pull from)
and local config  give the cpus to use

Reply:
@mult1fractal  yes we have 3 "shared" configs now.

the "container" config which is shared between docker and singularity. that's why docker or singularity true is written in the nextflow.config. Thus both can share the same config and we only need to maintain one container file.

the "local" config which is different from the "node.config". the node.config will be more detailed with cpus and ram and is used for cluster environments (where you need to specify this) e.g. cloud.

all the other statements are therefore written in nextflow.config so we can share certain config files (like container.config) across different profiles, thus reducing the amount of code to maintain.

Reply:
cpus should now also work better to limit the amount of process that will spawn I rewrote certain parts so that a --cores will limit certain process and threads are assigned correctly.